syntax = "proto3";

package envoy.extensions.quic.proof_source.v3;

import "udpa/annotations/status.proto";

option java_package = "io.envoyproxy.envoy.extensions.quic.proof_source.v3";
option java_outer_classname = "ProofSourceProto";
option java_multiple_files = true;
option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/quic/proof_source/v3;proof_sourcev3";
option (udpa.annotations.file_status).package_version_status = ACTIVE;

// [#protodoc-title: QUIC proof source config]
// [#extension: envoy.quic.proof_source.filter_chain]

// Configuration for the default QUIC proof source.
message ProofSourceConfig {
}
